.github/workflows/shopify-cli.yml

```diff
@@ -130,6 +130,33 @@ jobs:
       - name: ${{ matrix.target }}
         run: pnpm nx run-many --all --skip-nx-cache --target=${{ matrix.target }}
 
+  oclif-manifests:
+    name: Check OCLIF manifests
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+      - name: Setup deps
+        uses: ./.github/actions/setup-cli-deps
+        with:
+          node-version: 18.7.0
+      - name: Build
+        run: pnpm build
+      - name: Refresh manifests
+        run: pnpm refresh-manifests
+      - name: Check if there are changes
+        id: changes
+        uses: UnicornGlobal/has-changes-action@v1.0.12
```

Comment:
What about scoping this check to the manifests? Some commands like `yarn install` against the integration app might lead to file changes that are unrelated to the manifests.